portfolio/src/main/webapp/scripts/maps.js

```diff
@@ -0,0 +1,161 @@
+/** Fetches patent filing data from the server and displays it in a map. */
+function createPatentFilingMap() {
+  fetch('/patent-data').then(response => response.json()).then((patentFilings) => {
+	// Create a new StyledMapType object, passing it an array of styles,
+	// and the name to be displayed on the map type control.
+	const styledMapType = new google.maps.StyledMapType(
```

Comment:
You can declare this variable outside this function, that way it isn't recomputed each time this fetch statement is called or bloat the function itself.

Reply:
got it, I thought this looked kind of weird as well but this is also how the tutorial did it so I wasn't sure. It makes sense to take it outside though I agree.

Reply:
![image](https://user-images.githubusercontent.com/37886623/84202967-96239a80-aa76-11ea-963d-70d544a7f0ef.png)
I'm getting these errors after moving it outside the function. I think there might be a reason that the documentation shows to keep it inside the function, so I'll keep it that way for now.